velox/type/Variant.cpp

```diff
@@ -227,11 +243,13 @@ std::string variant::toJson(const TypePtr& type) const {
       std::string b{};
       b += "[";
       bool first = true;
+      uint32_t idx = 0;
+      auto arrayElementType = (type != nullptr) ? type->childAt(0) : nullptr;
```

Comment:
This can simply be `auto arrayElementType = type->childAt(0)` since we have the null check on top.